### DISCUSSION TITLE
Import Date data for annotation

### DISCUSSION THREAD

**Discussion by ertugruluyar-github** [aidenlx/obsidian-zotero#220] [Help]
Would be good if there is a import date data for each annotation, so I can see that "When this annotation imported to the Obsidian?".
![image](https://github.com/PKM-er/obsidian-zotlit/assets/70096810/c0c62ec3-5d4e-423f-9166-6532b5f48b6e)

**Comment by aidenlx** [comment]
Annotations carry their own timestamps in v2 — `zt.dateAdded` and `zt.dateModified` — so you can render when an annotation was made or last edited in Zotero.

There is no separate "imported to Obsidian at" stamp, and the v2 update model makes one less meaningful: the managed region is re-rendered from current Zotero data on each update rather than accumulating imports over time. Closing as outdated against that model; the Zotero-side timestamps cover the practical case.

Field reference: https://zotlit.aidenlx.site/docs/reference/templates/data

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
